app/Http/Services/CategoryService.php

```diff
@@ -9,8 +10,27 @@ public function __construct(Category $category) {
         $this->category = $category;
     }
 
+    public function getALlCategory(){
+        return  DB::table('categories')->paginate(10);
```

Comment:
chỉ cần $this->category->paginate(<number>); ko cần dùng DB, với số 10 nhét vào file config nha.